alt/asyncapi/src/commands/validate.ts

```diff
@@ -0,0 +1,26 @@
+import {Command, flags} from '@oclif/command'
+import { parse } from '@asyncapi/parser'
+import { SpecificationFile } from '../models/SpecificationFile'
+
+export default class Validate extends Command {
+  static description = 'validates an AsyncAPI file'
+
+  static flags = {
+    help: flags.help({char: 'h'}),
+  }
+
+  static args = [{name: 'file'}]
+
+  async run() {
+    const {args} = this.parse(Validate)
+
+    const file = new SpecificationFile(args.file || 'asyncapi.yaml')
```

Comment:
To match the current implementation, this should fall back to context when a file is not specified. Didn't want to get that far for this example.